src/include/hip/rpp_hip_common.hpp

```diff
@@ -2514,6 +2514,26 @@ __device__ __forceinline__ void rpp_hip_interpolate8_nearest_neighbor_pln1(T *sr
     rpp_hip_interpolate1_nearest_neighbor_pln1(srcPtr, srcStrideH, locPtrSrc_f16->f1[7], locPtrSrc_f16->f1[15], roiPtrSrc_i4, &(dst_f8->f1[7]));
 }
 
+template <typename T>
+__device__ __forceinline__ void rpp_hip_load1_glitch(T *srcPtr, uint srcStrideH, float locSrcX, float locSrcY, int4 *roiPtrSrc_i4, float *dst, int channelStride, int chn)
```

Comment:
if the function is only specific for glitch
suggest moving this inside the hip kernel file itself instead of adding in common header

Reply:
Done